### PR TITLE
[Documentation] Concepts: Locales

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | yes
| New docs?    | yes
| BC breaks?      | no
| Related tickets | partially #5275 
| License         | MIT

### CODE REVIEW

File: docs/book/locales.rst [hunk 1]
```diff
@@ -4,20 +4,27 @@
...
+To support multiple languages we are using **Locales** in **Sylius**. Locales are language codes standardized by the ISO 15897.
 
-* id
-* code
-* enabled
-* createdAt
-* updatedAt
+.. tip::
+
+    In the dev environment you can easily check what locale you are currently using in the Symfony debug toolbar.
```
Comment (dantleech) [inline docs/book/locales.rst]: End with `:` ?

File: docs/book/locales.rst [hunk 2]
```diff
@@ -4,20 +4,27 @@
...
+
+    .. image:: ../_images/toolbar.png
+        :align: center
+
+Base Locale
+-----------
+
+During the :doc:`installation </book/installation>` you have provided a default base locale. This is the language in which everything
```
Comment (dantleech) [inline docs/book/locales.rst]: `During the installation you provided a default base locale` ?

File: docs/book/locales.rst [hunk 3]
```diff
@@ -4,20 +4,27 @@
...
+in your system will be saved in the database - all the product names, texts on website, e-mails etc.
 
 Locale Context
 --------------
 
 With the default configuration, customers are able to change the store language in the frontend.
 
-To manage the currently used language, we use **LocaleContext**. You can always access it through ``sylius.context.locale`` id.
+To manage the currently used language, we use the **LocaleContext**. You can always access it through ``sylius.context.locale`` id in the container.
```
Comment (dantleech) [inline docs/book/locales.rst]: `... access it from the container with the ID ....` ?

File: docs/book/locales.rst [hunk 4]
```diff
@@ -62,20 +67,15 @@ You can easily modify this logic by overriding this component.
         }
     }
 
-To get all languages configured in the store, including the disabled ones, you can simply use the repository.
+To get all languages configured in the store, including the disabled ones, use the locales repository.
```
Comment (dantleech) [inline docs/book/locales.rst]: `:` colon?

**Comment by pjedrzejewski** [comment]
Thank you Magda!

**Pull request merged**